scripts/tools/Makefile

```diff
@@ -0,0 +1,22 @@
+# no default goal
+# .DEFAULT_GOAL :=
+
+.PHONY: ktw
+kwt:
+	./kwt.sh
+
+.PHONY: kwt-connect
+kwt-connect:
+	./kwt_connect.sh
+
+.PHONY: task
```

Comment:
I wouldn't put task/terminal/web to tools, but instead to something like `extra` or `plugins` (as they are internal, no external tooling provided)